studio/local_worker.py

```diff
@@ -261,11 +261,11 @@ def main(args=sys.argv):
     parsed_args, script_args = parser.parse_known_args(args)
     verbose = parse_verbosity(parsed_args.verbose)
 
-    worker_config = None
-    if parsed_args.config is not None:
-        print("Local Runner configuration file = {0}".format(parsed_args.config))
-        with open(parsed_args.config) as f:
-            worker_config = json.load(f)
+    # worker_config = None
+    # if parsed_args.config is not None:
+    #     print("Local Runner configuration file = {0}".format(parsed_args.config))
+    #     with open(parsed_args.config) as f:
+    #         worker_config = json.load(f)
```

Comment:
TODO: read in config file in any of possible formats (.json, .hocon, .yaml)
When we'll get to actually using it.